text/trace/0170-sampling-probability.md

```diff
@@ -617,10 +617,201 @@ stream of Span data with non-zero values in the `log_head_adjusted_count`
 field can approximately and accurately count Spans using adjusted
 counts.
 
-Non-probabilistic Samplers such as the [Leaky-bucket rate-limited
-sampler](https://github.com/open-telemetry/opentelemetry-specification/issues/1769)
-SHOULD set the `log_head_adjusted_count` field to zero to indicate an
-unknown adjusted count.
+### Span data model changes
+
+The OpenTelemetry trace data "model" is not currently specified in a
+stand-alone way, the way it has been done for logs and metrics.  This
+OTEP calls for the creation of a new `trace/datamodel.md` file to
+capture this sort of specification detail.
```

Comment:
Should this just be an issue in specification? Not sure why this should be in the OTEP. Also 110% agreeing with you on this.

Reply:
This OTEP needs to introduce around 10 new technical terms that are used within the SDK specification, so I'm adding the content I would insert related to sampling here, not specifying a complete data-model for spans.

Reply:
> This
OTEP calls for the creation of a new `trace/datamodel.md` file to
capture this sort of specification detail.

I feel that this is an issue that can be filed against specs.

Reply:
https://github.com/open-telemetry/opentelemetry-specification/issues/1929